### ISSUE TITLE
std/node require uses Deno.core.evalContext

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno#9066]
Deno.core.evalContext isn't a public API, so it should not be used by std. Could we replace it with a regular `eval`?

**Comment by bnoordhuis** [comment]
`eval()` won't show the right file name in exception stack traces. I doubt you can patch that up through `Error.prepareStackTrace`.

**Comment by kitsonk** [comment]
Agreed, there is always a potential need to evaluate in a given context.  Since it will never be part of the spec, should we consider proxying it in `Deno.evalContext()`?

**Comment by guybedford** [comment]
Does `sourceURL` work for eval in Deno?

**Comment by lucacasonato** [comment]
No. But good point, we should add support for that.
